### PR TITLE
Add Pact Broker Exporter

### PR DESCRIPTION
**Pull request by pperzyna**

### CODE REVIEW

**Comment by brian-brazil** [comment]
Can you add the DCO and keep it sorted please?

You should be using MustNewConstMetric for all your metrics, as is it won't handle pacts disappearing correctly.

**Comment by pperzyna** [comment]
@brian-brazil It toke me a while to fulfill the DCO and moved to MustNewConstMetric, but it is done.

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -197,6 +197,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Unbound exporter](https://github.com/kumina/unbound_exporter)
    * [WireGuard exporter](https://github.com/MindFlavor/prometheus_wireguard_exporter)
    * [Xen exporter](https://github.com/lovoo/xenstats_exporter)
+   * [Pact Broker exporter](https://github.com/ContainerSolutions/pactbroker_exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: Keep it sorted.

Comment (pperzyna) [reply content/docs/instrumenting/exporters.md]: done

**Comment by pperzyna** [comment]
@brian-brazil done

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**